accord-core/src/test/java/accord/utils/MessageTask.java

```diff
@@ -70,6 +71,7 @@ public String toString()
     private final List<Node.Id> recipients;
     private final String desc;
     private final Request request;
+    private final CommandStore commandStore;
```

Comment:
need to debug this class as it looks like its crossing stores often... ATM I try to avoid that as I added a check to block that case (this is the only section that hit it)